packages/mint-components/src/components/sqm-stat-container/sqm-stat-container-view.tsx

```diff
@@ -34,8 +35,26 @@ export function StatContainerView(props: StatContainerProps, children: VNode) {
         height: "100%",
       },
       "& > ::slotted(*:last-child)": {
-        "border-right": "1px solid #ffffff",
+        "border-right": "none",
       },
+      // TODO: Try Make this happen without media queries
+      // "@media screen and (max-width: 499px)": {
```

Comment:
Flagging code cleanup

Reply:
@zachharrison should we keep this commented out section?

Reply:
> @00salmon would also note I saw a lot of errors in the console when viewing in grapes. ![image](https://user-images.githubusercontent.com/50340998/154127482-fd68b39c-cfbc-4822-bf43-e19e45e9c892.png)

seems to be fixed!